core/src/main/groovy/com/novoda/gradle/release/BintrayConfiguration.groovy

```diff
@@ -21,6 +21,7 @@ class BintrayConfiguration {
             key = propertyFinder.getBintrayKey()
             publish = extension.autoPublish
             dryRun = propertyFinder.getDryRun()
+            override = propertyFinder.getOverride()
```

Comment:
Not  related to this PR but why don't we have just `propertyFinder.override` instead of the getter. It would look little bit nicer.